aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -151,12 +154,15 @@ def calcjob_outputcat(calcjob, path):
         )
 
     try:
-        content = retrieved.get_object_content(path)
+        # When we `cat`, it makes sense to directly send the output to stdout as it is
+        with retrieved.open(path, mode='rb') as fhandle:
+            copyfileobj(fhandle, sys.stdout.buffer)
     except (IOError, OSError) as exception:
```

Comment:
We can just catch OSError in python 3, as IOError is just an alias:
https://docs.python.org/3/library/exceptions.html#IOError

(see a few lines above the link)

Reply:
Yes, I was wondering why... Now py2 is dropped we can just catch OSError

Reply:
Yes, maybe just fix it here, but I opened #4082 to fix it in the whole codebase